modules/insights-operator-gather-duration.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+:_content-type: PROCEDURE
+[id="insights-operator-gather-duration_{context}"]
+= Viewing Insights Operator gather durations
+
+You can view the time it takes for the Insights Operator to gather the information contained in the archive. This helps in understanding Insights Operator resource usage and troubleshooting issues with Insights Advisor.
+
+
+.Perquisites 
+
+* A recent copy of your Insights Operator archive
```

Comment:
You could add a full stop at the end of this line, for consistency with the other prerequisite in the page.

Reply:
fixed